modules/create_timepoint/php/create_timepoint.class.inc

```diff
@@ -85,13 +85,13 @@ class Create_Timepoint extends \NDB_Form
         $user_list_of_projects = $user->getProjectIDs();
         if (count($user_list_of_projects) == 1) {
             //if there is only one project, autoselect first project from array of 1
-            //TODO: change this to array_key_first() when support is only PHP 7.3+
-            $project = \Project::getProjectFromID(array_pop($user_list_of_projects));
+            $project_id = $user_list_of_projects[0];
         } else if (count($user_list_of_projects) > 1) {
             $project_id = intval($values['project']);
-            $project    = \Project::getProjectFromID($project_id);
         }
 
+        $project = \Project::getProjectFromID($project_id);
```

Comment:
This looks like a change in behaviour? Why is this required for 7.3?

Reply:
other than that if it's passing Travis, it looks like changes are minimal so it's good enough for me.

Reply:
It's not required. It was an enhancement but I agree it's not very helpful here. I'll revert this file.